src/index.ts

```diff
@@ -13,13 +13,15 @@ if (!Utilities.hasDotEnvVars()) {
     process.exit(1)
 }
 
-console.log(Utilities.checkSatellitesPropsReliability())
-
 const prepareDatabase = async (): Promise<void> => {
     await dbConnectionProvider.connectToDatabase()
     const countDbSats = await SatelliteUtilities.getSateliteCount()
     const countFileSats = Utilities.countSatelitesFromFile()
-    if (countDbSats < countFileSats) SatelliteUtilities.removeCollectionIfExists().then(() => Utilities.prePopulateDatabase())
+    if (countDbSats < countFileSats) SatelliteUtilities.removeCollectionIfExists()
```

Comment:
1. `countFileSats` is the number of satellites in the file, but only some are going to be inserted into database (only the complete ones). I think it will be better to have `countFileSats` be the count of **valid/complete** satellites in file.
2. Change the condition to:
```ts
if (countDbSats !== countFileSats)
```
so that database is repopulated when the satellite count differs between DB and file. If you keep `<` condition, the database won't get repopulated after removing some satellites from the file.